packages/testing/config/jest/api/RedwoodApiJestEnv.js

```diff
@@ -0,0 +1,34 @@
+const NodeEnvironment = require('jest-environment-node')
+
+class RedwoodApiJestEnvironment extends NodeEnvironment {
+  constructor(config, context) {
+    super(config, context)
+    this.testPath = context.testPath
+  }
+
+  async setup() {
+    await super.setup()
+
+    // Disable per-request-context in testing.
+    process.env.SAFE_GLOBAL_CONTEXT = '1'
+
+    this.global.testPath = this.testPath
+  }
+
+  async teardown() {
+    await super.teardown()
+  }
+
+  getVmContext() {
+    return super.getVmContext()
+  }
+
+  // async handleTestEvent(event, state) {
+  //   if (event.name === 'test_start') {
+  //     // Link to event docs:
+  //     // https://github.com/facebook/jest/blob/master/packages/jest-types/src/Circus.ts
+  //   }
+  // }
```

Comment:
Is this an example for further discussion?

Reply:
No discussion needed. Left intentionally. It was a part of the Custom Env template. Danny and I worked on this together and found it helpful when attempting to merge this and setup.js into one file, although we didn't end up going in that direction. Just had the feeling we might need it in the future.